cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/FlowletTwillRunnable.java

```diff
@@ -51,9 +47,4 @@ public void handleCommand(Command command) throws Exception {
       LOG.debug("Command failure ignored", e);
     }
   }
-
-  @Override
-  protected Arguments resolveScope(Arguments arguments) {
```

Comment:
It was added in the previous log level PR #6705 , thought it is good to revert back. Will change it back if it is simpler.